templates/oam/traits/automaticsidecarinjection.Traefik.meshery.layer5io.schema.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please remove this file.

Reply:
Removed.
Now trait doesn't have any file.
Should there be any other trait for Traefik?

Reply:
I don't think there are any traits for traefik just yet.